File: src/TmrOverlay.App/Overlays/GapToLeader/GapToLeaderOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.GapToLeader;

internal static class GapToLeaderOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "gap-to-leader",
        DisplayName: "Gap To Leader",
        DefaultWidth: 560,
        DefaultHeight: 260,
        Options:
        [
            OverlaySettingsOptionDescriptor.Integer(
                OverlayOptionKeys.GapCarsAhead,
                "Cars ahead",
                0,
                12,
                defaultValue: 5),
            OverlaySettingsOptionDescriptor.Integer(
                OverlayOptionKeys.GapCarsBehind,
                "Cars behind",
                0,
                12,
                defaultValue: 5)
        ],
        ShowSessionFilters: false);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/GapToLeader/GapToLeaderOverlayDefinition.swift
import AppKit

enum GapToLeaderOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "gap-to-leader",
        displayName: "Gap To Leader",
        defaultSize: NSSize(width: 560, height: 260),
        showSessionFilters: false,
        fadeWhenLiveTelemetryUnavailable: true
    )
}
